shared/common-adapters/back-button.native.js

```diff
@@ -2,7 +2,9 @@
 
 import React, {Component} from 'react'
 import {NativeTouchableWithoutFeedback} from './native-wrappers.native'
-import {Badge, Box, Icon, iconCastPlatformStyles} from '.'
+import Badge from './badge'
```

Comment:
never import from common-adapters/index as that pulls in everything, including this file